pkg/resources/server.go

```diff
@@ -324,6 +365,9 @@ func (rs *resourceServer) cleanUp() error {
 	if err := rs.resourcePool.CleanDeviceInfoFile(rs.resourceNamePrefix); err != nil {
 		errors = append(errors, err.Error())
 	}
+	if err := rs.cdi.CleanupSpecs(); err != nil {
```

Comment:
i think we should:

1. cleanup spec files which are related only to this resource  server her
2. in the resource manager we should cleanup any remaining cdi spec files before starting servers just in case on un-gracefull exit of previous device plugin instance.

thoughts ?

Reply:
IMO, it's better to move this logic to resource manager only: it will cleanup all spec files before resorce servers start, so no raise condidion in this case and all outdated specs will be removed.  With such behaviour we don't need to filter for orphaned spec files if config changed during device plugin restart